scanpy/experimental/pl/_preprocessing.py

```diff
@@ -0,0 +1,111 @@
+from typing import Optional
+from anndata import AnnData
+from scanpy.plotting import scatter
+import numpy as np
+
+
+def pearson_residuals_hvg_scatter(
+    adata: AnnData,
+    marker_names=None,
+    gene_name_key: Optional[str] = None,
```

Comment:
please replace to `var_names `. In general, check the signature of already existing functions in order to harmonize arguments name.